SU-SWS/react_paragraphs
# D8CORE-1823 Fix link widget when the user inputs <front>

# READY FOR REVIEW

# Summary
- Fix the form when a user enters `<front>` to display that if they go back to edit the item.

# Need Review By (Date)
- 4/29

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. create a page with a card component
1. populate the link field with `<front>`
1. save and verify the url for the card goes to the home page
1. edit that page and edit the card you created
1. verify the link url says `<front>` and not `/<front>`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
